Application/Api/Reporting/Standard/Person/Person.php

```diff
@@ -267,18 +267,36 @@ public function downloadAgreementClassList($DivisionId = null)
      * @param null $Date
      * @param null $Type
      * @param string $DivisionName
+     * @param string $GroupName
      *
      * @return bool|string
      */
-    public function downloadAbsenceList($Date = null, $Type = null, $DivisionName = '')
+    public function downloadAbsenceList($Date = null, $Type = null, $DivisionName = '', $GroupName = '')
```

Comment:
Codacy found an issue: [Incorrect spacing between argument "$Date" and equals sign; expected 0 but found 1](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)